<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0,0,0">
     <item row="0" column="0" rowspan="3" colspan="9">
      <widget class="QLabel" name="image_label">
       <property name="styleSheet">
        <string notr="true">border: 2px solid black;
        background: white;</string>
       </property>
       <property name="text">
        <string>Изображение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QPushButton" name="add_button">
       <property name="minimumSize">
        <size>
         <width>129</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>129</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item row="8" column="8">
      <widget class="QLabel" name="volume_label">
       <property name="text">
        <string>20%</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="9">
      <widget class="QLabel" name="title_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid black;
        background: white;</string>
       </property>
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3" colspan="3">
      <widget class="QPushButton" name="edit_button">
       <property name="minimumSize">
        <size>
         <width>229</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>229</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="8">
      <widget class="QSlider" name="duration_slider">
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal {
        height: 8px;
        background: #c0c0c0;
        border-radius: 4px;
    }

    QSlider::handle:horizontal {
        background: #000000;
        border: 2px solid #000000;
        width: 15px;
        height: 10px;
        border-radius: 9px;
        margin: -8px 0;
    }
</string>
       </property>
       <property name="maximum">
        <number>1200</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="8">
      <widget class="QLabel" name="time_label">
       <property name="text">
        <string>0:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QPushButton" name="choose_right_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>→</string>
       </property>
      </widget>
     </item>
     <item row="8" column="7">
      <widget class="QSlider" name="loud_slider">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>71</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">

    QSlider::handle:vertical {
        background: #000000;
        border: 2px solid #000000;
        width: 15px;
        height: 10px;
        margin: -8px 0;
    }
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QPushButton" name="player_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>67</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> QPushButton {
        background: grey;
        color: white;
        border: none;
        border-radius: 20px;
        padding: 12px 24px;
        font-size: 16px;
        font-weight: bold;
    }
QPushButton:pressed {
    /* Эффект нажатия */
    background-color: #d0d0d0;
    border: 1px solid #6c6c6c;
    padding-top: 6px;  /* Смещение текста вниз */
    padding-bottom: 4px;
}</string>
       </property>
       <property name="text">
        <string> ▶ </string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="choose_left_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Громкость</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="6" colspan="3">
      <widget class="QPushButton" name="delete_button">
       <property name="minimumSize">
        <size>
         <width>129</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>129</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="9">
      <widget class="QLabel" name="author_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid black;
        background: white;</string>
       </property>
       <property name="text">
        <string>Автор:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="9">
      <widget class="QLabel" name="genre_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid black;
        background: white;</string>
       </property>
       <property name="text">
        <string>Жанр:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>10</y>
      <width>16</width>
      <height>561</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>160</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="just_label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="author_line">
       <property name="placeholderText">
        <string>Автор</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="title_line">
       <property name="placeholderText">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="genre_box"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Длительность:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Min:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="min_time_edit">
       <property name="maximumTime">
        <time>
         <hour>0</hour>
         <minute>20</minute>
         <second>0</second>
        </time>
       </property>
       <property name="displayFormat">
        <string>mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Max:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="max_time_edit">
       <property name="maximumTime">
        <time>
         <hour>0</hour>
         <minute>20</minute>
         <second>0</second>
        </time>
       </property>
       <property name="displayFormat">
        <string>mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="find_button">
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
